kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -157,6 +157,31 @@ public class ResponsiveConfig extends AbstractConfig {
   private static final String SUBPARTITION_HASHER_DOC = "Hasher to use for sub-partitioning.";
   private static final Class<?> SUBPARTITION_HASHER_DEFAULT = Murmur3Hasher.class;
 
+
+  // ------------------ WindowStore configurations ----------------------
+
+  public static final String WINDOW_BLOOM_FILTER_ENABLED_CONFIG = "responsive.window.bloom.filter.enabled";
+  private static final boolean WINDOW_BLOOM_FILTER_ENABLED_DEFAULT = false;
+  private static final String WINDOW_BLOOM_FILTER_ENABLED_DOC = "Whether to enable a bloom filter for "
```

Comment:
maybe we can make this an int config, and the value can represent the last x windows to maintain bloom filters for, and default to 0 and limit to 1 for now?

Reply:
Ack, will do in the followup PR